### ISSUE TITLE
consistent use of goundhogR vs goundhogr

### ISSUE THREAD

**Issue by urisohn** [groundhogR/groundhogR#40]
sometimes, we have uppercase, e.g., name of pacakge
sometimes lowercase, e.g., get.goundhogr.folder

It seems worthit it always use groundhogR. No?

**Comment by Bisaloo** [comment]
It's only a matter of style. The most proficient R developer nowadays tend to avoid uppercase in package names because it takes longer to type and it's more prone to errors ("Were some character uppercase? Which ones? I don't remember").

But there is no official guideline for this so both choices are possible.

**Comment by urisohn** [comment]
OK. I would keep the package as groundhogR.
I think the only function that has it differently is the groundhogr.folder related ones

**Comment by Bisaloo** [comment]
would it make more sense to rename it `get.groundhog.folder()` to match `groundhog.library()`?

**Comment by urisohn** [comment]
actually, yes. i like that.

do you think the package should just be {groundhog}

it will hurt with SEO, but is that the only downside?

**Comment by Bisaloo** [comment]
It's available on CRAN, bioconductor and GitHub so it's possible.

I think groundhogR is fine (although I slightly prefer groundhogr) but both options work well IMO.

**Comment by urisohn** [comment]
i prefer uppercase
but what do you think of no R, just {groundhog}

**Comment by Bisaloo** [comment]
Yeah, I understood you meant groundhog (no R) but I don't have a strong opinion. I prefer it to groundhogR but only because it's lowercase. Otherwise, I think it's nice to have a more distinctive name.

**Comment by urisohn** [comment]
ok, i will ask a few more people then.

**Comment by Bisaloo** [comment]
Relevant blog post: https://www.njtierney.com/post/2018/06/20/naming-things/

**Comment by Bisaloo** [comment]
So, should I rename everything with groundhog (no R)?

**Comment by urisohn** [comment]
i think it makes sense. I am conflcted, but i think groundhogR will lead to confusion when using groundhog.library() (no R) and when telling a colleague verbally to use groundhog it will be easy without an R.

If you have reservations though, i am very much on the fence.
